clearing_operations/scripts/PointTargetGRG.py

```diff
@@ -524,19 +538,43 @@ def main():
         #UPDATE
         targetLayerName = os.path.basename(outputFeatureClass)
         if appEnvironment == "ARCGIS_PRO":
-            results = arcpy.MakeFeatureLayer_management(outputFeatureClass, targetLayerName).getOutput(0)
-            mapList.addLayer(results, "AUTO_ARRANGE")
-            layer = findLayerByName(targetLayerName)
-            if (layer):
-                arcpy.AddMessage("Labeling grids")
-                labelFeatures(layer, gridField)
+            #params = arcpy.GetParameterInfo()
+            ## get the symbology from the GRG.lyr
+            #scriptPath = sys.path[0]
+            #layerFilePath = os.path.join(scriptPath,r"commondata\userdata\GRG.lyrx")
+            #arcpy.AddMessage("Applying Symbology from {0}".format(layerFilePath))
+            #params[8].symbology = layerFilePath
+            
+            arcpy.AddMessage("Do not apply symbology it will be done in the next task step")
+            
         elif appEnvironment == "ARCMAP":
-            layerToAdd = arcpy.mapping.Layer(outputFeatureClass)
-            arcpy.mapping.AddLayer(df, layerToAdd, "AUTO_ARRANGE")
-            layer = findLayerByName(targetLayerName)
-            if (layer):
-                arcpy.AddMessage("Labeling grids")
-                labelFeatures(layer, gridField)
+                           
+            arcpy.AddMessage("Adding features to map (" + str(targetLayerName) + ")...")
+            
+            arcpy.MakeFeatureLayer_management(outputFeatureClass, targetLayerName)
+            
+            # create a layer object
+            layer = arcpy.mapping.Layer(targetLayerName)            
+            
+            # get the symbology from the NumberedStructures.lyr
+            layerFilePath = os.path.join(os.getcwd(),"data\Layers\GRG.lyr")
```

Comment:
@adgiles - I believe it will return the path of where "python " was run from - but if you want to guarantee a path it might be better to use `__FILE__` (ex `os.path.dirname(__file__)`).

Also just to verify - is `GRG.lyr` included with this toolbox?